packages/clay-navbar/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+	entry: './src/ClayNavbar.js',
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /(node_modules|bower_components)/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						compact: false,
+						presets: ['babel-preset-es2015'],
+						plugins: ['babel-plugin-transform-node-env-inline'],
+					},
+				},
+			},
+		],
+	},
+	output: {
+		library: 'metal',
+		libraryTarget: 'this',
+		filename: './build/globals/clay-navbar.js',
+	},
+	plugins: [new webpack.optimize.ModuleConcatenationPlugin()],
+	resolve: {
+		mainFields: ['esnext:main', 'main'],
+		alias: {
+			'incremental-dom': path.resolve('./node_modules/incremental-dom'),
```

Comment:
We need to add `metal-soy` as in https://github.com/metal/metal-clay-components/pull/76